File: frontend/devTALK/src/DTO/SuggestionDTO.ts
export class SuggestionDTO {
	username: string;
	profilePicture: string;
	name: string;
	connectionSent: boolean;

	constructor(
		username: string,
		profilePicture: string,
		name: string,
		connectionSent: boolean = false,
	) {
		this.username = username;
		this.profilePicture = profilePicture;
		this.name = name;
		this.connectionSent = connectionSent;
	}
}

File: backend/devtalk/src/main/java/com/amakakeru/devtalk/DTO/SuggestionDTO.java
package com.amakakeru.devtalk.DTO;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class SuggestionDTO {
	private String username;
	private String profilePicture;
	private String name;
}